pep-0426.txt

```diff
@@ -1487,9 +1487,6 @@ justifications for needing such a standard can be found in PEP 386.
 
 .. _Python Package Index: http://pypi.python.org/pypi/
 
-.. [2] PEP 301:
-   http://www.python.org/dev/peps/pep-0301/
```

Comment:
I think it's fine to drop this - the internal details of PyPI aren't directly relevant to the PEP.